logs/logs.html

```diff
@@ -6,8 +6,17 @@
     </title>
   </head>
   <body>
-    <h1>Logs</h1>
+    <h2>Logs</h2>
     <hr>
-    <p>LogLogLog</p>
+    <h1>Team Log:</h1>
+    <p>10/16/18:All members present drove the robot prototype.</p>
+    <h1>Programmer log:</h1>
```

Comment:
I will just get to the point
```suggestion
    <h2>Programmer log:</h2>
```